Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'slim-rails'
 gem 'bootsnap', '>= 1.4.4', require: false
 # For testing purpouses adding FactoryBot
 gem 'factory_bot_rails'
+gem 'email_validator'
```

Comment:
Nice! Es ari kaut kur to izmantoju!

Reply:
Haha, pretty sure ka jaunākajos railos kkas tads jau ir iebūvēts :D + pretty sure, ka devise arī kkā šo valdiē, līdz ar to gan jau modelim šis nemaz nav vajadzīgs vispār :)

Reply:
Tomēr nav vēl iebūvējuši railos un dokumentācijā piedāvā:
```
class EmailValidator < ActiveModel::EachValidator
  def validate_each(record, attribute, value)
    unless value =~ /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
      record.errors.add attribute, (options[:message] || "is not an email")
    end
  end
end

class Person < ApplicationRecord
  validates :email, presence: true, email: true
end
```

Reply:
Starp citu pilnais email regexp ir vnk crazy, dzīvē reti kad tādu vajag